promgen/views.py

```diff
@@ -1033,8 +1033,15 @@ def get_queryset(self):
             return self.queryset.filter(
                 Q(alertlabel__name="Service", alertlabel__value__contains=search)
                 | Q(alertlabel__name="Project", alertlabel__value__contains=search)
+                | Q(alertlabel__name="Job", alertlabel__value__contains=search)
```

Comment:
May be better to do `alertlabel__value__icontains` so that upper/lower case do not matter

Reply:
fixed